src/main/java/me/botsko/prism/database/mysql/MySQLPrismDataSource.java

```diff
@@ -2,47 +2,68 @@
 
 import com.zaxxer.hikari.HikariConfig;
 import com.zaxxer.hikari.HikariDataSource;
-import me.botsko.prism.database.sql.*;
+import me.botsko.prism.Prism;
 import me.botsko.prism.database.SelectQuery;
+import me.botsko.prism.database.sql.SQLPrismDataSource;
+import me.botsko.prism.database.sql.SQLSelectQueryBuilder;
 import org.bukkit.configuration.ConfigurationSection;
 
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.HashMap;
+
 /**
  * Created for use for the Add5tar MC Minecraft server
  * Created by benjamincharlton on 5/04/2019.
  */
 public class MySQLPrismDataSource extends SQLPrismDataSource {
 
-    private boolean nonStandardSQL;
-    private static HikariConfig dbconfig = new HikariConfig();
+    private Boolean nonStandardSql;
+    public static final HashMap<String, String> dbInfo = new HashMap<>();
+    private static HikariConfig dbConfig = new HikariConfig();
 
+    /**
+     * Create a dataSource.
+     *
+     * @param section Config
+     */
     public MySQLPrismDataSource(ConfigurationSection section) {
         super(section);
-        nonStandardSQL = this.section.getBoolean("useNonStandardSql", true);
+        nonStandardSql = this.section.getBoolean("useNonStandardSql", false);
+        detectNonStandardSql();
         name = "mysql";
     }
 
+    /**
+     * Add default params to new Config.
+     * //todo Version the config so we can remove the ones we dont need.
+     *
+     * @param section Config
+     */
     public static void updateDefaultConfig(ConfigurationSection section) {
         section.addDefault("hostname", "127.0.0.1");
         section.addDefault("username", "root");
         section.addDefault("password", "");
         section.addDefault("databaseName", "minecraft");
         section.addDefault("prefix", "prism_");
         section.addDefault("port", "3306");
-        section.addDefault("useNonStandardSql", true);
+        //section.addDefault("useNonStandardSql", true);
```

Comment:
Instead of commenting this out, I'd just remove it. No point keeping commented code, it's in git history if you want to restore it.